spec/models/flyover_comments/vote_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+module FlyoverComments
+  RSpec.describe Vote, :type => :model do
+    pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
After moving the upvote/downvote increment/decrement stuff into the vote model, we should test it here (upvote creation, downvote creation, updating a vote to change the value, upvote destruction, downvote destruction cases)